identifiers/country-us/census_autogenerated/us_sldu.csv

```diff
@@ -1953,6 +1959,23 @@ ocd-division/country:us/territory:pr/sldu:5,Puerto Rico State Senate district V
 ocd-division/country:us/territory:pr/sldu:6,Puerto Rico State Senate district VI
 ocd-division/country:us/territory:pr/sldu:7,Puerto Rico State Senate district VII
 ocd-division/country:us/territory:pr/sldu:8,Puerto Rico State Senate district VIII
+ocd-division/country:us/territory:as/sldu:1,American Samoa State Senate district 1
+ocd-division/country:us/territory:as/sldu:2,American Samoa State Senate district 2
+ocd-division/country:us/territory:as/sldu:3,American Samoa State Senate district 3
+ocd-division/country:us/territory:as/sldu:4,American Samoa State Senate district 4
+ocd-division/country:us/territory:as/sldu:5,American Samoa State Senate district 5
+ocd-division/country:us/territory:as/sldu:6,American Samoa State Senate district 6
+ocd-division/country:us/territory:as/sldu:7,American Samoa State Senate district 7
+ocd-division/country:us/territory:as/sldu:8,American Samoa State Senate district 8
+ocd-division/country:us/territory:as/sldu:9,American Samoa State Senate district 9
+ocd-division/country:us/territory:as/sldu:10,American Samoa State Senate district 10
+ocd-division/country:us/territory:as/sldu:11,American Samoa State Senate district 11
+ocd-division/country:us/territory:as/sldu:12,American Samoa State Senate district 12
+ocd-division/country:us/territory:mp/sldu:1,Northern Mariana Islands State Senate district 1
+ocd-division/country:us/territory:mp/sldu:2,Northern Mariana Islands State Senate district 2
+ocd-division/country:us/territory:mp/sldu:3,Northern Mariana Islands State Senate district 3
+ocd-division/country:us/territory:vi/sldu:st_croix,U.S. Virgin Islands Senate St. Croix District

```

Comment:
Isn't there an at-large district as well for the US Virgin Islands?

Reply:
Yes, there is one like this but that is considered a congressional district rather than an sldu, I believe: ocd-division/country:us/territory:vi/cd:at-large